vaccine_feed_ingest/runners/us/giscorps_vaccine_providers/normalize.py

```diff
@@ -280,10 +223,10 @@ def normalize_state_name(name: str) -> str:
     else:
         name = name.lower().capitalize()
 
-    if name in us_state_abbrev:
-        return us_state_abbrev[name]
-
-    return name.upper()
+    try:
+        return us.states.lookup(name).abbr
+    except Exception:
```

Comment:
`Exception` is too broad of an exception class. Can you catch the specific exception, or check for `None` directly